controller/src/main/java/io/pravega/controller/server/security/auth/AuthorizationResource.java

```diff
@@ -46,10 +42,7 @@ public static String ofScopes() {
      * @throws NullPointerException if {@code scopeName} is null
      * @throws IllegalArgumentException if {@code scopeName} is empty
      */
-    public static String ofScope(String scopeName) {
-        Exceptions.checkNotNullOrEmpty(scopeName, "scopeName");
-        return scopeName;
-    }
+    String ofScope(String scopeName);
```

Comment:
is there any specific advantage in making these methods non static and accessing it on object?